app/src/main/java/com/ataulm/whatsnext/account/SignInActivity.kt

```diff
@@ -47,13 +47,8 @@ class SignInActivity : BaseActivity() {
         }
     }
 
-    override fun onStart() {
-        super.onStart()
-        presenter!!.startPresenting()
-    }
-
-    override fun onStop() {
-        presenter!!.stopPresenting()
-        super.onStop()
+    override fun onDestroy() {
```

Comment:
onStart/Stop were probably the wrong lifecycle methods to use: just because an activity is being backgrounded doesn't mean that we don't want any request that's being made to get cancelled. On the contrary, we'd expect that work to continue and the screen to update when that activity is returned to the foreground.

onDestroy, we can call `cancel` on the coroutine scope because we know that this presenter will no longer be used (once a scope's job is cancelled, no further coroutines can run on it).